client/app/billing/autoRenew/bulk/translations/Messages_fr_FR.json

```
@@ -1,5 +1,6 @@
 {
   "billing_autorenew_bulk": "Vous avez sélectionné {{ count }} services",
+  "billing_autorenew_bulk_one": "Vous avez sélectionné 1 service",
```

```suggestion
  "billing_autorenew_bulk_one": "Vous avez sélectionné un service",
```
In French you are supposed to write in letters numbers between 0 and 9 but this is nitpicking :)